### DISCUSSION TITLE
Referencing components on multiple diagrams

### DISCUSSION THREAD

**Discussion by bladnor** [jgraph/drawio-desktop#1836] [General]
Hi
Is it possible to create components and reuse them by reference (not a copy) on multiple diagrams? For example, when I change the description or the color of an element, it has to change on all diagrams where the component is used.
I found this information (see link below 1*) which tells that it is not possible with the confluence plugin:
"._..Unfortunatel, the feature you are requesting can't be realized due to limitations that the Confluence macro brings along for app vendors ..._".
So how is it possible with the desktop version?

Thanks Roland

1]
https://community.atlassian.com/t5/Confluence-questions/Re-Re-Reuse-a-draw-io-object-not-the-diagram-in-two-d/qaq-p/2283927/comment-id/265481#M265481

**Comment by davidjgraph** [comment]
It's not currently possible on the desktop tool. It would be better suited to implement on app.diagrams.net out of all the platforms, but it's not on our roadmap currently.
